test/e2e/customconfigs/managejobswithoutqueuename_test.go

```diff
@@ -410,3 +413,80 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("ManageJobsWithoutQueueName without JobSet integration", ginkgo.Ordered, func() {
+	var (
+		ns           *corev1.Namespace
+		defaultRf    *kueue.ResourceFlavor
+		localQueue   *kueue.LocalQueue
+		clusterQueue *kueue.ClusterQueue
+	)
+
+	ginkgo.BeforeAll(func() {
+		updateKueueConfiguration(func(cfg *config.Configuration) {
+			cfg.ManageJobsWithoutQueueName = true
+			cfg.Integrations.Frameworks = slices.Filter(nil, cfg.Integrations.Frameworks, func(framework string) bool {
+				return framework != jobset.FrameworkName
+			})
+		})
+	})
+
+	ginkgo.BeforeEach(func() {
+		ns = util.CreateNamespaceFromPrefixWithLog(ctx, k8sClient, "e2e-")
+		defaultRf = testing.MakeResourceFlavor("default").Obj()
+		gomega.Expect(k8sClient.Create(ctx, defaultRf)).Should(gomega.Succeed())
+		clusterQueue = testing.MakeClusterQueue("cluster-queue").
+			ResourceGroup(
+				*testing.MakeFlavorQuotas(defaultRf.Name).
+					Resource(corev1.ResourceCPU, "2").
+					Resource(corev1.ResourceMemory, "2G").Obj()).Obj()
+		gomega.Expect(k8sClient.Create(ctx, clusterQueue)).Should(gomega.Succeed())
+		localQueue = testing.MakeLocalQueue("main", ns.Name).ClusterQueue("cluster-queue").Obj()
+		gomega.Expect(k8sClient.Create(ctx, localQueue)).Should(gomega.Succeed())
+	})
+
+	ginkgo.AfterEach(func() {
+		gomega.Expect(util.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
+		util.ExpectObjectToBeDeletedWithTimeout(ctx, k8sClient, clusterQueue, true, util.LongTimeout)
+		util.ExpectObjectToBeDeletedWithTimeout(ctx, k8sClient, defaultRf, true, util.LongTimeout)
+		util.ExpectAllPodsInNamespaceDeleted(ctx, k8sClient, ns)
+	})
+
+	ginkgo.When("manageJobsWithoutQueueName=true", func() {
+		ginkgo.It("should create only one workload for parent job", func() {
+			aw := awtesting.MakeAppWrapper("aw", ns.Name).
+				Queue(localQueue.Name).
+				Component(testingjobset.MakeJobSet("job-set", ns.Name).
+					ReplicatedJobs(
+						testingjobset.ReplicatedJobRequirements{
+							Name:        "replicated-job-1",
+							Replicas:    2,
+							Parallelism: 2,
+							Completions: 2,
+							Image:       util.E2eTestAgnHostImage,
+							Args:        util.BehaviorWaitForDeletion,
+						},
+					).
+					SetTypeMeta().
+					Suspend(false).
+					RequestAndLimit("replicated-job-1", corev1.ResourceCPU, "200m").
+					Obj()).
+				Suspend(false).
+				Obj()
+
+			ginkgo.By("creating an unsuspended appwrapper without a queue name", func() {
+				gomega.Expect(k8sClient.Create(ctx, aw)).To(gomega.Succeed())
+			})
+
+			ginkgo.By("check that only one workload is created and admitted", func() {
```

Comment:
Would this test actually fail without the fix? The issue description actually says the `appwrapper-app-wrapper-2deaa` workload is created and admitted. Maybe we could instead lower the number of Pods, and check the Job gets finished? I suppose it would never get finished before the fixes, right?

Reply:
But you right. This is not stable test need to optimize it.

Reply:
Done